modules/end-user-guide/examples/che-ref_a-kubernetes-secret-as-a-file.adoc

```diff
@@ -86,5 +85,5 @@ components:
        containerPath: /my/new/path
    ...
 ----
-+
-Note that for this kind of overrides, components must declare an alias to be able to distinguish containers which belong to them and apply override path exclusively for those containers.
+
+NOTE: With overrides, an alias declared in the components will distinguish the containers for the override, and the override path will apply only to those containers.
```

Comment:
I'm not sure what this note is saying. I recommend addressing the user in the second person and telling them what they should expect to see and do.

IBM SG: "Tense: Write in the simple present tense as much as possible. Use past or future tense only when you cannot use present tense or it does not make sense to use present tense."

Also, "distinguish" is a cognitive process. It is typically performed by a person or animal, and is often used with "from". For example, in the active voice, "You can usually distinguish butterflies from moths by their greater size and variety of color."   Or, in the passive voice, "Butterflies and moths can typically be distinguished from each other by their size and variety off color." To make this previous example sound more antique and literary, I can rewrite it as,  "Their size and variety of color will distinguish butterflies from moths." However, this last example has no place in technical writing.

Reply:
This is work in progress. I committed these changes, but I have a different draft open.
By the way, I'm only planning fixing this sentence, because I opened a new issue for this:
https://issues.redhat.com/browse/RHDEVDOCS-3166

Reply:
Would you mind having a second look? I double-checked the relevant statements within the original file, so it should be factually correct now.